### ISSUE TITLE
Window.MobileAccessibility.usePreferredTextZoom(true)  does not work for IPAD - IOS

### ISSUE THREAD

**Issue by christie1phl** [phonegap/phonegap-mobile-accessibility#52]
Window.MobileAccessibility.usePreferredTextZoom(true)  does not work for IPAD - IOS. Please help

**Comment by dpa99c** [comment]
You need to set the preferred content mode on iPadOS to mobile in `config.xml`:

`<preference name="PreferredContentMode" value="mobile" />` 

see the [Cordova docs for `config.xml`](https://cordova.apache.org/docs/en/10.x/config_ref/)

**Comment by shamis** [comment]
hi @dpa99c ,

have made this change but still on ipad the font size does not change. Have you made any other changes to make it work ?
```xml
<platform name="ios">
        <preference name="PreferredContentMode" value="mobile" />
        <allow-intent href="itms:*" />
```
even trying to use `setTextZoom` does nothing on the ipad
TIA
